src/main/java/com/basho/riak/client/core/RiakNode.java

```diff
@@ -136,7 +136,7 @@ public void operationComplete(ChannelFuture future) throws Exception
                 // the next time it's pulled from the pool.
                 // We record the disco for the health check.
                 recentlyClosed.add(new ChannelWithIdleTime(future.channel()));
-                logger.error("inAvailable channel closed; id:{} {}:{}",
+                logger.warn("Available channel closed; id:{} {}:{}",
```

Comment:
It could be normal, or could signify something bigger is going on with the network / application if seen in conjunction with other errors, or many of them grouped together.

https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html#WARN